### DISCUSSION TITLE
doris单张表存5亿条用户数据，是否合适？

### DISCUSSION THREAD

**Discussion by 15374604475** [apache/doris#14503] [RFC]
背景：业务需要5亿用户的档案数据，目前计划存放在doris中，供后端查询
痛点：是否支持高并发查询如此亿级别数据
优化方案：将用户数据存储到hbase中，请问还有其他优化建议吗？
欢迎大家讨论

**Comment by kpfly** [comment]
多大的并发？档案数据schema什么样

**Reply by 15374604475** [reply]
500左右并发，schema大约30个字段，大多是varchar及datetime类型

**Comment by kpfly** [comment]
varchar最大多大

**Reply by 15374604475** [reply]
最大不超过100，档案信息都是短字符

**Reply by kpfly** [reply]
用doris非常合适，不用hbase

**Reply by 15374604475** [reply]
请教下，5亿的数据量所支持的并发量是根据何种维度去判断呢？类似这种数据是否有必要加上分区呢

**Reply by caoliang-web** [reply]
这种还是要合理的设置分区和分桶的
